helper/tlsutil/config.go

```diff
@@ -41,6 +43,34 @@ var supportedTLSCiphers = map[string]uint16{
 	"TLS_RSA_WITH_AES_256_CBC_SHA":            tls.TLS_RSA_WITH_AES_256_CBC_SHA,
 }
 
+// algorithmStringRepr is the string representation of a signing algorithm
+type algorithmStringRepr string
```

Comment:
why does this need a type alias here?

Reply:
This is so that the set of possible elements is strongly typed. As there are only two elements it might not achieve that much with a type alias, but I find when groups of elements are finite having a type alias to represent that can be helpful.